learning_resources/constants.py

```diff
@@ -52,11 +52,11 @@ class OfferedBy(ExtendedEnum):
     """
 
     mitx = "MITx"
-    ocw = "OCW"
+    ocw = "MIT OpenCourseWare"
     bootcamps = "Bootcamps"
     xpro = "xPRO"
```

Comment:
We might want to change this while we're here. I'll try to confirm. 
```suggestion
    xpro = "MIT xPRO"
```

Reply:
Okay. When you're able to confirm it update the issue and I'll change it.

Reply:
Confirmed and I updated the issue. Sorry again for this oversight.